### PR TITLE
change Kube-ovn to Kube-OVN and fix grammar issues

### PR DESCRIPTION
**Pull request by oilbeater**
Pull Request Description
---

### Describe what this PR does / why we need it
Change Kube-ovn to Kube-OVN and fix grammar issues
### Does this pull request fix one issue?

No
### Describe how you did it
No
### Describe how to verify it
Doc change
### Special notes for reviews

Also make some clarification that Kube-OVN has the default VPC and Subnet that can be used by workloads in all namespaces. Even if users have no idea about the VPC and the Subnet, the default settings will make all things work just like Flannel.

### CODE REVIEW

File: docs/other-implementations.md [hunk 1]
```diff
@@ -4,30 +4,30 @@ There has already so many of CNI implementations in the open source community, w
...
-and widely-adapted. Without multi-tenant network support, Hybridnet takes more efforts to adapt to user's exist underlay network
+For the overall design, unlike Kube-OVN's offering rich functions and features, Hybridnet is always designed to be convenient
+and widely-adapted. Without multi-tenant network support, Hybridnet takes more effort to adapt to the user's existing underlay network
 and always follows the [Kubernetes network model](https://kubernetes.io/docs/concepts/cluster-administration/networking/#the-kubernetes-network-model).
 For example, when underlay and overlay network exist in one Hybridnet cluster at the same time (hybrid mode), every underlay
-pod will also get a overlay "identity" with the same ip address automatically while communicating with overlay pods, to
+pod will also get an overlay "identity" with the same ip address automatically while communicating with overlay pods, to
 ensure the full network connectivity within the cluster.
 
 One of the biggest differences is that Kube-ovn uses OVN/OVS as the dataplane, while Hybridnet uses common networking
```
Comment (mars1024) [inline docs/other-implementations.md:19]: Another missed `Kube-ovn`

**Comment by mars1024** [comment]
@oilbeater Welcome! Hope Kube-OVN and hybridnet will have more interactions later.

File: docs/other-implementations.md (hunk 1 above)
Comment (oilbeater) [reply docs/other-implementations.md]: fixed

**Review by mars1024 — APPROVED** [review]
/lgtm

**Comment by mars1024** [comment]
@oilbeater can you sign the commit following [https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification) ?

**Comment by oilbeater** [comment]
@mars1024 commit signed

**Review by mars1024 — APPROVED** [review]

**Pull request merged**